web/js/components/util/image-crop.js

```diff
@@ -94,6 +96,7 @@ Crop.defaultProps = {
   height: 10,
   maxHeight: window.innerWidth,
   maxWidth: window.innerHeight,
+  keepSelection: false,
```

Comment:
We don't always want it to be false, because in the case of smart-handoffs, we need to override and set it to true. This makes it so the bounding box doesn't disappear when a user targets something else in the UI. If I only place `keepSelection={false}`  when I instantiate the component within smart-handoffs.js, it doesn't recognize it. Thus, the reason for me adding it in. I added the default state to not change the behavior for gifs and snapshots.

Reply:
Ah, yep.  I see what's happening now.  Looks good 👍